scenes/game/local_game.gd

```diff
@@ -2555,9 +2579,16 @@ func is_effect_condition_met(performing_player : Player, effect, local_condition
 		elif condition == "initiated_at_range":
 			var range_min = effect['range_min']
 			var range_max = effect['range_max']
-			var initiated_strike = active_strike.initiator == performing_player
+			if active_strike.initiator != performing_player:
+				return false
+
+			# check each space that the opponent occupied
 			var starting_distance = active_strike.starting_distance
-			return initiated_strike and starting_distance >= range_min and starting_distance <= range_max
+			var other_size = 1 + (2 * other_player.width)
```

Comment:
I feel like maybe this should just be calling a function to determine something.
Like, width should only be handled within the Player class as if it were a private variable?